dockers/texcmp/texcmp.js

```diff
@@ -16,18 +15,12 @@ var readFile = Q.denodeify(fs.readFile);
 var writeFile = Q.denodeify(fs.writeFile);
 var mkdir = Q.denodeify(fs.mkdir);
 
-// ignore some tests, since they contain commands not supported by LaTeX
-var blacklist = {
```

Comment:
I feel like this is more closely related to the test case than to the application running it. Or, perhaps phrased better, like this is more like configuration data not executable code, so it should go into the configuration file not the implementation file. When someone creates a new test case as a copy of an existing one, it makes sense to copy any special behavior as well. It is true that these settings get included in the query parameter, but having more data in there doesn't hurt anyone. I also like the idea of using the value of the mapping for a human-readable explanation of the reason why this should be blacklisted, and having it in JS entails many of the drawbacks of JSON, as exemplified by the doubled backslashes below. Simply writing that reason with little concern for syntax, perhaps even over multiple lines, makes it more readable in my opinion.

Have I convinced you, @xymostech?